knowledge-base/kb-gridview-howto-increase-size-checkbox-selectcolumn.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Increase the CheckBox Size of the GridViewSelectColumn
+description: Make the CheckBox displayed in the GridViewSelectColumn larger
+type: how-to
+page_title: Enlarge the CheckBox shown by the GridViewSelectColumn
+slug: kb-gridview-howto-increase-size-checkbox-selectcolumn
+position: 0
+tags: gridview, selectcolumn, increase, checkbox, size
+ticketid: 1403577
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product Version</td>
+	    	<td>2018.3.1016</td>
+	    </tr>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>RadGridView for WPF</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to increase the CheckBox size of the GridViewSelectColumn.
+
+## Solution
+
+1. Define the following style targetting the **CheckBox** control.
+
+  #### __[XAML]__
+  {{region xaml-kb-gridview-howto-increase-size-checkbox-selectcolumn-1}}
+    <Style x:Key="LargeCheckBoxStyle" TargetType="CheckBox" BasedOn="{StaticResource CheckBoxStyle}">
```

Comment:
We can leave a note that you should base on the default style only, if you are using the NoXaml binaries.